chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1920,10 +1920,19 @@ public void setTargetSpp(int value) {
   }
 
   /**
+   * @param modified Whether to alter the branch count based on current spp
```

Comment:
This boolean changes the behavior of the whole method. Can we create a new method instead? `getCurrentBranchCount` maybe, with the same explanation?

Reply:
Done.